Update README (#46)
codus/codelog: README.md

[![Maintainability](https://api.codeclimate.com/v1/badges/6f5885536c6b5c82f304/maintainability)](https://codeclimate.com/github/codus/codelog/maintainability)
[![Test Coverage](https://api.codeclimate.com/v1/badges/6f5885536c6b5c82f304/test_coverage)](https://codeclimate.com/github/codus/codelog/test_coverage)

> A gem to help big teams manage their chagelogs.

Reviewer: typo -> `chagelogs`

Reviewer: Also, I might be  wrong but, i think the gem does a bit more than helping big teams managing their changelogs.

Author: ```suggestion
A gem to help big teams manage their changelogs.
```

Once all changes were merged and the release is ready to be packed, all that must be done is to run the following command:

```bash
$ codelog release [VERSION] <RELEASE_DATE>
```

Where the `VERSION` is mandatory and represents the number of the next version of the system. An additional parameter `RELEASE_DATE` can be used, and if so, the release date of the version will be that value. You can configure the format of this parameter in the configuration file, under `changelogs/codelog.yml`.

This command will execute 3 steps:

- Generates a new release file at `changelogs/releases/` by merging all change files at `changelogs/unreleased/`

> - Deletes the change files at `changelogs/unreleased/` because they now compose the new release.(if these files are not deleted, the same changes would appear in the next release.

Reviewer: Looks like the parentheses are being opened but are not being closed on line 99. I would also change the `if these files are not deleted` to `Otherwise`

Author: ```suggestion
- Deletes the change files at `changelogs/unreleased/` because they now compose the new release.(Otherwise, the same changes would appear in the next release).
```
